src/sqlfluff/dialects/dialect_postgres.py

```diff
@@ -1335,6 +1335,106 @@ class AlterTableActionSegment(BaseSegment):
     )
 
 
+@postgres_dialect.segment()
+class CreateMaterializedViewStatementSegment(BaseSegment):
+    """A `CREATE MATERIALIZED VIEW` statement.
+
+    As specified in https://www.postgresql.org/docs/13/sql-creatematerializedview.html
```

Comment:
Worth updating these doc links to postgres 14 and checking the grammar matches that so we're up to date 👍